src/SDK.ts

```diff
@@ -436,6 +436,23 @@ export default class SDK implements ISDK {
 
     const data: InitContext = initContext || {};
 
+    var cachObj = {
+      "orgId":this.omnichannelConfiguration.orgId,
+      "widgetId":this.omnichannelConfiguration.widgetId
+    }
+
+    if (data && data.customContextData != null) {
+      var tempArr = this.sortCustomContextData(data.customContextData);
+      Object.assign(cachObj, {"customContext": tempArr});
+    }
+
+    if (data.portalcontactid != "" && data.portalcontactid != null) {
+        Object.assign(cachObj, {"portalcontactid": data.portalcontactid});
+    }
+
+    var hash = require('crypto');
```

Comment:
Import should be on top of the file. We shouldn't use `require`, and should use `import` instead